src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
@perploug so if i log out of my account and someone logs back in, anything we put into local storage will remember, we should be careful about what we put in there and the potential problems it could cause. If someone doesn't have access to this node, will it get an error? what will happen? (i.e. a JS error occurs and the user cannot continue as a worst case)